NEWS.md

```diff
@@ -4,12 +4,14 @@
 * `NA`s are allowed again for the `placename` or `latitude`/`longitude` arguments (also empty strings for `placename`). 
   These queries are not sent to the API. 
   An empty/`NA` response will be returned depending on the specific query ([#143](https://github.com/ropensci/opencage/issues/143)).
+* `opencage_forward()`, `opencage_reverse()`, and `opencage_key()` are now deprecated (leveling up from soft-deprecated, [#147](https://github.com/ropensci/opencage/pull/147)).
```

Comment:
oh the note is here!

Reply:
do you know of https://github.com/cynkra/fledge? One interesting aspect in particular is the support for conventional commits.

Reply:
sorry, that was confusing. I was just lazy and didn't want to resolve git conflicts. 😄 
Yes, I know fledge. 

> Using fledge is a discipline / a few habits that is worth learning!

Not sure, I am _that_ disciplined. Case in point ☝🏻 😉